packages/editor/src/components/post-publish-button/index.js

```diff
@@ -66,17 +78,38 @@ export class PostPublishButton extends Component {
 			onSave();
 		};
 
+		const buttonProps = {
+			'aria-disabled': isButtonDisabled,
+			className: 'editor-post-publish-button',
+			isBusy: isSaving && isPublished,
+			isLarge: true,
+			isPrimary: true,
+			onClick,
+		};
+
+		const toggleProps = {
+			'aria-disabled': isToggleDisabled,
+			'aria-expanded': isOpen,
+			className: 'editor-post-publish-panel__toggle',
```

Comment:
Should we update this to be `editor-post-publish-toggle` instead? We can maintain both the existing and the new class until the toggle component is removed as to preserve compatibility. I could prepare a follow-up PR to this one with the changes if we wanted to do it.

Reply:
Or perhaps a better alternative would be to remove the `editor-post-publish-panel__toggle` class once the toggle component is removed.

Reply:
`editor-post-publish-panel__toggle` seems like the right name.